ds/set/set.go

```diff
@@ -167,7 +168,7 @@ func (s *Set) SUnion(keys ...string) (val [][]byte) {
 // SDiff Returns the members of the set resulting from the difference between the first set and all the successive sets.
 func (s *Set) SDiff(keys ...string) (val [][]byte) {
 
-	if len(keys) < 2 || !s.exist(keys[0]) {
+	if !s.exist(keys[0]) {
```

Comment:
It will panic if the `keys` is empty. You should check the slice length before reading the first element.

Here is the test case to prove:
```go
t.Run("empty key", func(t *testing.T) {
	var keySet []string
	members := set.SDiff(keySet...)
})
```

Reply:
Yeah，you're right. It's an easy mistake to make.